content/en/error_tracking/frontend/mobile/reactnative.md

```diff
@@ -5,4 +5,914 @@ code_lang: reactnative
+
+-   JavaScript source maps for your iOS JavaScript bundle
+-   JavaScript source maps for your Android JavaScript bundle
+-   dSYMs for your iOS native code
+-   Proguard mapping files if you have enabled code obfuscation for your Android native code
+
+To set your project up to send the symbolication files automatically, run `npx datadog-react-native-wizard`.
+
+See the wizard [official documentation][8] for options.
+
+#### How to configure deobfuscation
+
+{{% collapse-content title="Using Datadog Metro Configuration" level="h5" %}}
+
+Starting from `@datadog/mobile-react-native@2.10.0` and `@datadog/datadog-ci@v3.13.0`, the SDK exports a Datadog Metro Plugin, which attaches a unique Debug ID to your application bundle and sourcemap.
+
+Add it to your `metro.config.js` to allow for accurate symbolication of stacktraces on Datadog:
+
+```js
+const {getDefaultConfig, mergeConfig} = require('@react-native/metro-config');
+const {withDatadogMetroConfig} = require('@datadog/mobile-react-native/metro');
+
+// Your configuration
+const config = mergeConfig(getDefaultConfig(__dirname), {});
+
+module.exports = withDatadogMetroConfig(config);
+```
+{{% /collapse-content %}}
+
+
+{{% collapse-content title="Using the `datadog-ci react-native inject-debug-id` command" level="h5" %}}
+
+As an alternative to the Metro Configuration, starting from `@datadog/mobile-react-native@2.10.0` and `@datadog/datadog-ci@v3.13.0`, you can use the `datadog-ci react-native inject-debug-id` command to manually attach a unique Debug ID to your application bundle and sourcemap.
+
+Usage instructions are available on the [command documentation page][201].
+
+[201]: https://github.com/DataDog/datadog-ci/blob/master/src/commands/react-native/README.md#inject-debug-id
+
+{{% /collapse-content %}}
+
+
+{{% collapse-content title="Passing options for your uploads" level="h5" %}}
+##### Using the `datadog-sourcemaps.gradle` script
+
+To specify a different service name, add the following code to your `android/app/build.gradle` file, before the `apply from: "../../node_modules/@datadog/mobile-react-native/datadog-sourcemaps.gradle"` line:
+
+```groovy
+project.ext.datadog = [
+    serviceName: "com.my.custom.service"
+]
+```
+
+##### Using the `datadog-ci react-native xcode` command
+
+Options for the `datadog-ci react-native xcode` command are available on the [command documentation page][301].
+
+##### Specifying a custom release version
+
+Use the `DATADOG_RELEASE_VERSION` environment variable to specify a different release version for your source maps, starting from `@datadog/mobile-react-native@2.3.5` and `@datadog/datadog-ci@v2.37.0`.
+
+When the SDK is initialized with a version suffix, you must manually override the release version in order for the source map and build versions to match.
+
+[301]: https://github.com/DataDog/datadog-ci/tree/master/src/commands/react-native#xcode
```

Comment:
This link leads to a `404` error, and I'm not sure what to update it with since the folder structure of that repo has changed.